generators/server/templates/src/main/java/package/config/WebConfigurer.java.ejs

```diff
@@ -91,6 +94,9 @@ import java.util.concurrent.TimeUnit;
 /**
  * Configuration of web application with Servlet 3.0 APIs.
  */
+<%_ if (messageBrokerKafka) { _%>
+@EnableBinding({KafkaSseConsumer.class, KafkaSseProducer.class})
+<%_ } _%>
```

Comment:
Not so happy to have moved this one here, but permits to disable the async configuration in tests while keeping kafka up and running